test/unit-test/mock/IClientMock.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * ******************************************************************************
+ * Copyright (c) 2019 Robert Bosch GmbH.
```

Comment:
is this file new, or moved from somewhere else? if it is new as the diff seems to think, maybe update to 2020 and instead of Bosch write your or your companies  name. In case it is modified/copied from an existing one feel free to add credentials. Nothing is also fine as long as the License note is kept.
Was just puzzled to see a "new" file that seems to have time travellled from Bosch 2019 😄

Reply:
> is this file new, or moved from somewhere else? 

It's new. update in 3e67c23. 

> instead of Bosch write your or your companies name

I am working in Bosch :blush:. Hope can chat with you in future.